mod/calculated_metrics/spec/lib/formula/calculator/input_item/company_option/company_query_spec.rb

```diff
@@ -0,0 +1,218 @@
+# frozen_string_literal: true
+
+RSpec.describe Formula::Calculator::InputItem::Options::CompanyOption::CompanySearch::CompanyQuery do
+  let(:more_evil_id) { Card.fetch_id "Jedi+more evil" }
+  let(:less_evil_id) { Card.fetch_id "Jedi+less evil" }
+  let(:supplied_by_id) { Card.fetch_id "Commons+Supplied by" }
+  let(:deadliness_id) { Card.fetch_id "Jedi+deadliness" }
+
+  describe "#sql", output_length: 5000 do
+    def sql str
+      described_class.new(str, nil).sql.tr("\n", " ").squeeze(" ").strip
+    end
+
+    def eq_sql sql
+      eq sql.strip_heredoc.tr("\n", " ").squeeze(" ").strip
```

Comment:
wondering if we should `gsub(/\s+/, " ")` or something (on both sides) so changes in white space don't break tests.

Reply:
that's what `squeeze` does